src/Build.UnitTests/BackEnd/TaskHostFactory_Tests.cs

```diff
@@ -61,10 +61,21 @@ public void TaskNodesDieAfterBuild()
             }
         }
 
-        [Theory]
+        [DotNetOnlyTheory]
+        [InlineData(false)]
+        // [InlineData(true)] <-- explicitly opting out on core will lead to node crash
+        public void VariousParameterTypesCanBeTransmittedToAndReceivedFromTaskHost_NetCore(
```

Comment:
NIT: Is it still Theory with just one InlineData

Reply:
Good point.
I wanted to preserve the argument (ond overall the same signature as for the FulFw test) to make it explicit that we do not want to test the legacy behavior for core (to prevent anyone from attempting to reverse add-it).